composer.json

```diff
@@ -39,6 +41,7 @@
     "psr/http-message": "To use standard GraphQL server"
   },
   "scripts": {
-    "lint" : "phpcs"
+    "lint" : "phpcs",
+    "static-analysis": "@php ./vendor/bin/phpstan.phar analyse --ansi -l 1 -c phpstan.neon src"
```

Comment:
why use phar instead of standard binary?

`composer require --dev phpstan/phpstan` and then `vendor/bin/phpstan`

Reply:
cause phar has less dependencies and this lib don't likes to have useless requirements

Reply:
They're dev requirements so it doesn't really matter. Also, whether they're hidden in phar file or in vendor folder doesn't make much difference.

However, adding it via composer is more flexible (eg. easier to add phpstan phpunit or strict rules)

Reply:
I'll change it :+1: